packages/node/src/relay/index.ts

```diff
@@ -39,6 +39,10 @@ export default class RelayClient extends BaseSession {
   }
 
   private _handleSignals(): void {
+    if (process.env.IGNORE_SIGNALS) {
+      return logger.info('Ignoring SIGINT/SIGTERM as requested.')
+    }
```

Comment:
@lpradovera return if IGNORE_SIGNALS is set so the SDK does not attach listeners with `process.on`